optuna/study.py

```diff
@@ -326,8 +326,13 @@ def set_system_attr(self, key, value):
 
         self._storage.set_study_system_attr(self._study_id, key, value)
 
-    def trials_dataframe(self, include_internal_fields=False, multi_index=False):
-        # type: (bool, bool) -> pd.DataFrame
+    def trials_dataframe(
+        self,
+        include_internal_fields=False,  # type: bool
+        exclude_fields=('intermediate_values', ),  # type: Tuple[str]
```

Comment:
I think "attribute" or "attr" is a more common terminology in Python than "fields". What do you think about `exclude_attr(s)` or `exclude_attributes`?

Reply:
I agree with you. [This Python official documentation](https://docs.python.org/3/tutorial/classes.html#classes) also use `attribute`, and Optuna will be more pythonic if we follow the convention.

> Each class instance can have attributes attached to it for maintaining its state.

In terms of the choice of `attr` or `attribute`, I think `attr` is suitable because we have already used `attr` in `Trial` and `Study` like `Trial.system_attrs` and `Trial.user_attrs`.

To make the interface coherent, do we also change the name of `include_internal_fields` to `include_internal_attrs`?

Reply:
NOTICE: The following proposal is just an idea.

IMO, it's a bit difficult to imagine the effect of `include_internal_fields` and `exclude_fields`.
How about removing the two options and introducing a new `include_attrs` option?
The new option explicitly specifies the columns (and the order) included in the resulting data frame (i.e., the defautl value is `include_attrs=('number', 'state', 'value', 'datetime_start', 'datetime_complete', 'params', 'user_attrs', 'system_attrs')`).

Reply:
@sile 
Thank you for your proposal. I also concerned about the interdependency of `include_internal_fields` and `exclude_fields`.  If we have both of them, we need to think about the priority of those arguments, and it will make API complicated.

`include_attrs` seems to resolve the issue, and I'll apply it to this PR.

Reply:
I implemented the option as `attrs`. I simplified the name, but it may be less informative. Please let me know if you find other candidate names.